cmd/instance_show.go

```diff
@@ -76,6 +77,7 @@ If you wish to use a custom format, the available fields are:
 		ow.AppendDataWithLabel("initial_user", instance.InitialUser, "Initial User")
 		ow.AppendDataWithLabel("initial_password", instance.InitialPassword, "Initial Password")
 		ow.AppendDataWithLabel("ssh_key", instance.SSHKey, "SSH Key")
+		ow.AppendDataWithLabel("ssh_key_id", instance.SSHKeyID, "SSH Key ID")
```

Comment:
why do we need `ssh_key_id` if we already have `ssh_key`?

Reply:
I think we don't need `ssh_key ` just the `ssh_key_id`